app/utils/text.ts

```diff
@@ -0,0 +1,6 @@
+export function slugify(string: string, sep: string = '-'): string {
+    return string.toLowerCase()
+        .replace(/[^\w\s-]/g, '') // remove non-word [a-z0-9_], non-whitespace, non-hyphen characters
+        .replace(/[\s_-]+/g, sep) // swap any length of whitespace, underscore, hyphen characters with a single -
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5730106)